.github/workflows/build.yml

```diff
@@ -0,0 +1,198 @@
+name: Build
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    strategy:
+      matrix:
+        program:
+        - higan-ui
+        - byuu
+        - icarus
+        os:
+        - name: ubuntu
+          version: latest
+        - name: windows
+          version: latest
+        - name: macos
+          version: latest
+    runs-on: ${{ matrix.os.name }}-${{ matrix.os.version }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dependencies
+      if: matrix.os.name == 'ubuntu'
+      run: |
+        sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu `lsb_release -sc` main universe restricted multiverse"
+        sudo apt-get update -y -qq
+        sudo apt-get install libsdl2-dev libgtksourceview2.0-dev libgtk2.0-dev libao-dev libopenal-dev
+    - name: Make
+      run: make -j4 -C ${{ matrix.program }} build=performance local=false
```

Comment:
Any particular reason for 4? Does GitHub guarantee 4 CPUs? Did you test a bunch of values an discover that 4 was most efficient?

Reply:
Non-Mac workflows run in Azure Standard_DS2_v2 machines, which have 2 vCPUs. I picked 4 as a reasonable amount. I suspect the macOS runners might have more threads based on their performance, too.